### ISSUE TITLE
Element ordering in Statement: finetuning

### ISSUE THREAD

**Issue by davidzbiral** [DISSINET/InkVisitor#1527]
Thanks for implementing. Some requests:
- Props should appear in the natural order: source node, proptype, propvalue. Now it is propvalue, proptype - source node at the end with note.
- Only move up / down is really inconvenient - please implement DnD move. Including both those already sorted, and those not yet sorted.
- Perhaps explain the icon with lines and cross through mouseover? I presume, it is "remove order". And the icon with + would be "add order".

**Comment by Ptrhnk** [comment]
@davidzbiral 
1) We did that for readability but can be switched back
2) Try dragging anywhere by row, excluding buttons and tags (it was possible because there's no row click)
3) Yes

**Comment by davidzbiral** [comment]
@Ptrhnk : 
Ad 1: yes please, switch back. In props, include the source node as well.
Ad 2: great job!
Ad 3: OK.

**Comment by Ptrhnk** [comment]
@davidzbiral 
Ad 1: Do you think it'll be still clear what is what when it's gonna be just 3 tags next to each other?

**Comment by davidzbiral** [comment]
@Ptrhnk : Yes, if the fact that it is a prop is more visually prominent. I would consider, as an intro to the column, some icons or so. Now you have colours at the end - not bad, but I would make it visually more prominent. This is not urgent though, definitely not needed on Monday. The order origin - proptype - propvalue is important.

We are using a similar structure already in Detail, this one should be similar:
![obrazek](https://user-images.githubusercontent.com/55400794/213873076-e28b8636-e6f4-4278-9811-35bdee9d5b7a.png)

Of course, there, it is with headings on what it is, but I think we are fine as long as we have a strong visual trigger telling us that it is a prop (and not a relation, action, or actant).

**Comment by davidzbiral** [comment]
Hey @Ptrhnk, @adammertel : thinking about it, I think you were right and the order is nice! Sorry, I take back what I wrote. I began to like having the keyword (propvalue) at the first place. Just we need to signal better what the other two elements are, i.e. origin and especially proptype, which now appears in the opposite order than normally (normal order in a prop is origin - propvalue - proptype).

**Comment by adammertel** [comment]
@davidzbiral, thanks for your suggestions and your time with this new feature. 
ad 1. Yes, when creating the graphical design, we tried to prioritize the readability in the sense of rereading the statement sentence just by moving your eyes top-bottom through the table.
In contrast, the "used in" "meta property" table was more about understanding the structure of the statement and the position of a specific entity within. 
Maybe we should have communicated it more with historians before the implementation, but this time the deadline was very close; therefore, we skipped this step.

ad 2. the if this is not intuitive, we can implement the dragging handler as it is done, e.g., in the statement list
![image](https://user-images.githubusercontent.com/12932677/213925915-bc696648-b316-41bc-a72c-77cde1a70441.png)
